diablo/api/auth_controller.py

```diff
@@ -34,11 +33,16 @@
 
 
 @app.route('/api/auth/cas_login_url', methods=['GET'])
-@cache.cached()
 def cas_login_url():
     target_url = request.referrer or None
+    cas_url = _cas_client(target_url).get_login_url()
+    app.logger.info(f"""
+        cas_login_url for UID {current_user.uid}:
```

Comment:
We expect `current_user.uid` to be null but this will verify we're not redirecting authenticated sessions.